actions/aggregate.json

```diff
@@ -0,0 +1 @@
+{"ns":"sbtest.orders","op":"command","command":{"aggregate":"orders","pipeline":[{"$match":{"status":"A"}},{"$group":{"_id":"$cust_id","total":{"$sum":"$amount"}}},{"$sort":{"total":-1}}],"cursor":{},"$db":"sbtest"}}
```

Comment:
I guess we should not have those files outside of testdata?

Reply:
+1